docs/template_plugin/tests/functional/op_reference/convert.cpp

```diff
@@ -0,0 +1,438 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+
+struct ConvertParams {
+    template <class IT, class OT>
+    ConvertParams(const ngraph::PartialShape& shape, const ngraph::element::Type& iType, const ngraph::element::Type& oType, const std::vector<IT> iValues,
+                  const std::vector<OT> oValues, size_t iSize = 0, size_t oSize = 0) {
```

Comment:
IMHO, `iValues` and `oValues` doesn't need to be copy.

Reply:
Fixed